controllers/adminController.js

```diff
@@ -0,0 +1,33 @@
+const adminService = require('../services/adminService')
+
+const adminController = {
+  signIn: async (req, res, next) => {
+    const { email, password } = req.body
+    try {
+      const data = await adminService.signIn(email, password)
+      return res.status(200).json(data)
```

Comment:
```suggestion
      return res.json(data)
```
這邊與下面兩個 controller 可以改一下，因為 response 預設值就是 200，不用特別設定。